src/_nebari/upgrade.py

```diff
@@ -685,6 +685,26 @@ def _version_specific_upgrade(
         return config
 
 
+class Upgrade_2024_1_1(UpgradeStep):
+    version = "2024.1.1"
+
+    def _version_specific_upgrade(
+        self, config, start_version, config_filename: Path, *args, **kwargs
+    ):
+        rich.print("\n ⚠️  Warning ⚠️")
+        rich.print(
+            "-> Please run the [green]rm -rf stages[/green] so that we can regenerate an updated set of Terraform scripts for your deployment."
+        )
+        rich.print("\n ⚠️  Deprecation Warning ⚠️")
+        rich.print(
+            "-> jupyterlab-videochat, retrolab, jupyter-tensorboard and jupyter-nvdashboard",
```

Comment:
@dcmcand do you think we should add `jupyterlab-conda-store` here? We removed it from the `jupyterlab` Docker image.

Reply:
I wasn't sure on that one, but I guess we probably should.